LRBv21.py

```diff
@@ -332,11 +333,15 @@ def LRBv21(constants,radios,d,SparRange,
             # nu0 and cz0 guesses are from Lengyel which depends on an estimate of Tu using qpllu0
             # This means we cannot make a more clever guess for qpllu0 based on cz0 or nu0
             qpllu0_guess = si.qpllu0
-            qradial_guess = qpllu0_guess / np.trapz(si.Btot[si.Xpoint:] / si.Btot[si.Xpoint], x = si.S[si.Xpoint:])
+            # qradial_guess = qpllu0_guess / np.trapz(si.Btot[si.Xpoint:] / si.Btot[si.Xpoint], x = si.S[si.Xpoint:])
+            qradial_guess = (qpllu0_guess / si.Btot[si.Xpoint]) / np.trapz(1/si.Btot[si.Xpoint:], x = si.S[si.Xpoint:])
             st.cvar = 1/qradial_guess 
             
         # Initial guess of qpllt, the virtual target temperature (typically 0). 
-        st.qpllt = si.gamma_sheath/2*si.nu0*st.Tu*si.echarge*np.sqrt(2*si.Tt*si.echarge/si.mi)
+        if zero_qpllt is True:
+            st.qpllt = si.qpllu0 * 1e-2
```

Comment:
This is to force target heat flux to a low value (1% of the input). The sheath method sometimes led to 10% heat flux leaving the model. This makes things a bit less stable but more accurate.